### PR TITLE
Chinese localization added

### PR DESCRIPTION
**Pull request by okooo5km**
屏幕快照 2019-03-19 20 21 07

### CODE REVIEW

File: src/i18n/zh.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        join: '论坛'
+    },
+    nav: {
+        code: '开发',
+        cloud: '云服务',
+        docs: '文档'
+    },
+    bio: {
+        first: '基于 Apple <a target="_blank" href="https://github.com/apple/swift-nio">SwiftNIO</a> 构建的非阻塞、事件驱动架构，实现<a target="_blank" href="https://medium.com/@codevapor/vapor-3-0-0-released-8356fa619a5d">高性能</a>',
+        second: '使用 <a target="_blank" href="https://swift.org">Swift</a> 编程语言实现, 这门强大的语言简单易学',
```
Comment (Litreily) [inline src/i18n/zh.js:16]: Replace with "使用 <a target="_blank" href="https://swift.org">Swift</a> 这门强大而又简单易学的编程语言实现" should be better.

Comment (okooo5km) [reply src/i18n/zh.js:16]:
```suggestion
        second: '使用 <a target="_blank" href="https://swift.org">Swift</a> 这门强大而又简单易学的编程语言实现',
```

**Review by Litreily — APPROVED** [review]
That's useful for me to access this website in chinese. Good job!

**Review by tanner0101 — APPROVED** [review]
Thanks @smslit for submitting this and thanks @Litreily for the third-party review. This can be merged now. :)

**Comment by penny-coin** [comment]
Hey @smslit, you just merged a pull request, have a coin! 

You now have 1 coins.

**Comment by okooo5km** [comment]
Thanks for the coin

**Pull request merged**